src/scripts/promoRightSlider.js

```diff
@@ -13,19 +13,15 @@ rightPromoSlider.style.transform = 'translateX(' + -size * counter + 'px)';
 // event listeners
 
 nextBtn.addEventListener('click', () => {
-  if (counter >= rightPromoSliderImg.length - 1) {
-
-  } else {
+  if (!(counter >= rightPromoSliderImg.length - 1)) {
```

Comment:
no nie :P czy jak zechcesz odwrócić warunek `if (5 > 0)` to zrobisz

`if (!(5 > 0)` czy po prostu `if (5 <= 0)` ?